packages/@atjson/offset-annotations/src/annotations/data-set.ts

```diff
@@ -0,0 +1,44 @@
+import { BlockAnnotation, JSON } from "@atjson/document";
+
+export enum ColumnType {
+  PERITEXT = "peritext",
```

Comment:
So I know we only support `"peritext"` right now, but it may be good to point towards other types that will be supported, specifically, `"string"`, `"number"`, `"date"`?

Also... so we can get on the same page, is `ColumnType` referring to the type of the `"jsonValue"`?

Reply:
yes, it's referring to the parsing method that produces the jsonValue